peachjam_api/urls.py

```diff
@@ -12,6 +12,7 @@
 router.register(r"relationships", views.RelationshipViewSet, basename="relationships")
 router.register(r"works", views.WorksViewSet, basename="works")
 router.register(r"citation-links", views.CitationLinkViewSet, basename="citation-links")
+router.register(r"annotations", views.AnnotationViewSet, basename="annotations")
```

Comment:
this will have to be scoped to the document so the annotations can be loaded dynamically.

eg. `documents/<pk>/annotations`

Reply:
See how we do it in indigo - it's a clean and understandable URL pattern for nested REST resources: https://github.com/laws-africa/indigo/blob/master/indigo_api/urls.py#L14